### DISCUSSION TITLE
Support for 1.18+

### DISCUSSION THREAD

**Discussion by burn333** [mircokroon/minecraft-world-downloader#285] [Q&A]
I tried downloading the 1.15 pre-release to download my 1.18 world and the exe won't open at all.

**Comment by Peregrine05** [comment]
It's not fully released yet. That means there may still be issues, such as the one that you encountered.

**Comment by JorelAli** [comment]
@burn333 Feel free to join ongoing 1.18.x support discussions in #355.
